### PR TITLE
Spec controller

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
### The following project requirements have been implemented in this mile stone of the project:

Add RSpec as a gem to your project.

Create a Request spec file for all your controllers.

Make sure that for each action you check:

If response status was correct.

If a correct template was rendered.

If the response body includes correct placeholder text.

### CODE REVIEW

**Review by rustam-tolipov — COMMENTED** [review]
Hi @Ngufor-emmanuella, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that all `> Describe` messages should be removed. Kindly remove this [`Describe`](https://github.com/Ngufor-emmanuella/proj-blog-app/tree/spec-controller#:~:text=Describe%20how%20a%20new%20developer%20could%20make%20use%20of%20your%20project.) from your Readme and try to resubmit your code!

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Ngufor-emmanuella** [comment]
Hello thank you for your review but please I don't understand it. what > describe do you mean please is

File: README.md [hunk 1]
```diff
@@ -1,25 +1,190 @@
...
+- **Create a post for the application users**
+- **Interact with the posts (Like post or comment on Post)**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (bobb-Rob) [inline README.md]: - [ ] You should remove this comment as it is **NOT** intended for visitors of your repository but to guide you to edit this README correctly.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Ngufor-emmanuella, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**